lib/openhab/scripting.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# This is the default 'require' file in a Gemfile
+
+return if defined?(RSpec)
```

Comment:
what about if you use `bin/console`? is there a way we can reasonably detect the other way around - `return unless <running inside of real openHAB`?